src/hearings/containers/request-hearing/hearing-view-edit-summary/hearing-view-edit-summary.component.spec.ts

```diff
@@ -66,6 +66,19 @@ describe('HearingViewEditSummaryComponent', () => {
       expect(component.validationErrors.length).toEqual(1);
     });
 
+    it('should call removeUnnecessarySummaryTemplateItems in ngOnInit', () => {
```

Comment:
I really think you need a test that actually tests the functionality.  Refactor to make the template and screen flow constructor parameters (with defaults). 
Define a mock template, and mock screen flow, and pass these to the component under test. Check that the expected items are removed and that no unexpected items are removed